src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3064,273 +2973,263 @@
   "botania.page.speedUpBelt1": "If you go slow you're a Planeswalker",
 
-  "botania.entry.baubleBox": "Bauble Case",
-  "botania.tagline.baubleBox": "A handy container to store your Baubles",
-  "botania.page.baubleBox0": "Somtimes having too many $(4)Baubles$(0) to pick from can be a real problem, as inventory space is limited. A better way would be to store all of these in a $(1)Bauble Case$(0), a handy container for any $(4)Baubles$(0).$(p)The case's interface allows for quick switching of the current equipped baubles. The case can also store $(4)Mana$(0) containing items like a $(1)Mana Tablet$(0).",
+  "botania.entry.baubleBox": "Trinket Case",
+  "botania.tagline.baubleBox": "A handy container to store your Trinkets in",
+  "botania.page.baubleBox0": "Sometimes having too many $(4)Trinkets$(0) on hand is a problem-- inventory space is limited, after all. A solution: stash them in a $(1)Trinket Case$(0), a handy container for $(4)Trinkets$(0).$(p)The case's interface lets its user quickly swap their equipped $(4)Trinkets$(0); the case can also store $(4)Mana$(0)-containing items, like $(1)Mana Tablets$(0) (though they can't give/take $(4)Mana$(0) in that state).",
   "botania.page.baubleBox1": "It could even be a boat!",
 
   "botania.entry.dodgeRing": "Ring of Dexterous Motion",
   "botania.tagline.dodgeRing": "Sweep to the side to dodge attacks",
-  "botania.page.dodgeRing0": "The $(1)Ring of Dexterous Motion$(0) is a handy bauble to help prevent damage during combat. While worn, one can press their $(4)left/right$(0) keys (which are $(4)A and D$(0), respectively, by default) to quickly sweep to the side and dodge incoming attacks or mobs.$(p)The dodge ability goes through a short cooldown before it can be used again, and uses up some of the user's hunger bar.",
+  "botania.page.dodgeRing0": "The $(1)Ring of Dexterous Motion$(0) is a handy $(4)Trinket$(0) for dodging damage during duels. Its wearer can double-tap $(k:left) or $(k:right) to hurl themselves to the side and dodge incoming attacks/mobs.$(p)Dodging has a short cooldown, and burns some of its user's hunger.",
   "botania.page.dodgeRing1": "Dexterity bonus",
 
   "botania.entry.invisibilityCloak": "Invisibility Cloak",
   "botania.tagline.invisibilityCloak": "Exactly what it says on the tin",
-  "botania.page.invisibilityCloak0": "Not much explanation is required for the $(1)Invisibility Cloak$(0).$(p)It's a bauble worn on the $(4)body$(0) slot, that will take $(4)Mana$(0) from the wearer's inventory to make them, and their baubles invisible as long as it's worn and the cost is fulfilled.",
-  "botania.page.invisibilityCloak1": "You're a wizard, steve",
+  "botania.page.invisibilityCloak0": "No need to explain the $(1)Invisibility Cloak$(0).$(p)You wear it on your $(4)Body$(0) slot. It takes $(4)Mana$(0) from your inventory. It makes you and your $(4)Trinkets$(0) invisible.$(p)Symbiosis.",
```

Comment:
Nothing against references and jokes, but this one is still really out of place, thrown out at the end without rhyme and reason.